submenu-flow-js.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Navigation Example</title>
+  <link rel="stylesheet" href="node_modules/normalize.css/normalize.css">
+  <link rel="stylesheet" href="css/base.css">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+</head>
+<body class="nav-list-example">
+
+  <header role="banner">
+    <h1>Navigation list launching submenus and notifications demo</h1>
+  </header>
+
+  <div class="flexContainer flexSpaceAround">
+    <div class="col">
+      <nav class="pf-c-vertical-nav">
+        <ul class="pf-c-vertical-nav__content" role="list">
+          <li class="pf-c-vertical-nav__item">
+            <a href="#" class="pf-c-vertical-nav__link">
+              <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+              <span class="pf-c-vertical-nav__link-text">Home</span>
+            </a>
+          </li>
+
+          <li class="pf-c-vertical-nav__item">
+            <a href="#" class="pf-c-vertical-nav__link">
+              <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+              <span class="pf-c-vertical-nav__link-text">Technology</span>
+            </a>
+          </li>
+
+          <li class="pf-c-vertical-nav__item">
+            <a
+              href="#"
+              aria-expanded="false"
+              aria-controls="navbarSubmenu01"
```

Comment:
When testing this myself, it seems like `aria-controls` is best used when the element being controlled is not the next element in the DOM (e.g. it's far removed from the controlling element). A lot of information is announced by JAWS when it encounters this attribute, and it's only announced if the element being controlled is visible in the DOM. So the effect is that the information isn't announced until the user clicks the menu toggle, and then at that point it just seems like a lot of unnecessary noise.